addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1418,6 +1418,9 @@ body.editor_enable.editor_has_snippets {
                         background-color: $o-we-sidebar-content-field-input-bg;
                         color: inherit;
                         font-family: $o-we-sidebar-content-field-input-font-family;
+                        &:placeholder-shown {
+                            outline: 1px solid red;
+                        }
```

Comment:
Note that I suggested that implementation (to remove some js code and keep it simple) but since the placeholder on non-db link don't have a placeholder anymore (as qsm said broken in this PR and to fix), that didn't leave a very good impression as the purpose was to make it easily understandable to figure why you can't click on the toggle: "I can't click, which makes sense since the related input is red, exactly as a form submit".

But testing it, there is no red highlight on empty custom link since no placeholder anymore, making the click on the toggle hard to understand why it refuses to toggle.

Reply:
The placeholders are fixed (custom & DB links). 
OK, let's go back to the behavior of 15.2 here, the user can toggle an element whose input is empty and that redirects to '/'. :+1: